pom.xml

```diff
@@ -37,10 +37,10 @@
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<version.httpcomponents.fluent.hc>4.5.13</version.httpcomponents.fluent.hc>
 		<version.httpcomponents.httpclient>4.5.13</version.httpcomponents.httpclient>
-		<version.httpcomponents.httpcore>4.4.13</version.httpcomponents.httpcore>
-		<version.commons.io>2.5</version.commons.io>
-		<version.commons.codec>1.11</version.commons.codec>
-		<version.jackson.databind>2.9.10.7</version.jackson.databind>
+		<version.httpcomponents.httpcore>4.4.15</version.httpcomponents.httpcore>
```

Comment:
I don't think we need to update this library - there's no vulnerability for it. Plus, the `httpclient` dependency is relying on the older `httpcore` version.